### DISCUSSION TITLE
windows install

### DISCUSSION THREAD

**Discussion by Kc9jjj** [Yubico/libfido2#792] [Q&A]
Hi, i'm trying to figure out how to install libfido2 under windows 11. i'm wanting to use it for openssh but can't figure out how. thanks

**Comment by LDVG** [comment]
Hi,

From your question, it's not entirely clear to me what issues you're running into.  Microsoft provides a fork of OpenSSH that includes support for FIDO authenticators, see [PowerShell/Win32-OpenSSH](https://github.com/PowerShell/Win32-OpenSSH) for additional information and install instructions (the simplest likely being using `winget`).
